DenisZackharov/teacherss
# CRUD for subjects

### Summary

None.

### How it works

For director:
<img width="1423" alt="image" src="https://user-images.githubusercontent.com/78980031/230778791-c5f4032b-87d6-4e2e-a15d-f46c1dc2eac1.png">

For teacher:
<img width="1435" alt="image" src="https://user-images.githubusercontent.com/78980031/230778815-5cf18f35-6740-454c-bcfc-1c0a918d76db.png">


### Test plan

1. Login as director with credentials: `director@example.com` `123456`
2. Click to organization name `School 13`
3. Click`Subjects`
4. You can create new subject, edit and destroy existed

1. Login as teacher with credentials: `teacher@example.com` `123456`
2. Click to organization name `School 13`
3. Click`Subjects`
4. You can see existed subjects

# Checklist:


### Deploy notes

None.
